### ISSUE TITLE
unstable playback

### ISSUE THREAD

**Issue by treakiandroid** [badaix/snapcast#248]
```
2017-06-06 00-00-02 [out] onResync (default): 2840ms
2017-06-06 00-00-44 [out] onResync (default): 519ms
2017-06-06 00-01-19 [out] onResync (default): 3187ms
2017-06-06 00-01-57 [out] onResync (default): 110ms
2017-06-06 00-01-59 [out] onResync (default): 589ms
2017-06-06 00-02-34 [out] onResync (default): 97ms
2017-06-06 00-03-09 [out] onResync (default): 599ms
2017-06-06 00-03-11 [out] onResync (default): 310ms
2017-06-06 00-03-45 [out] onResync (default): 686ms
2017-06-06 00-04-56 [out] onResync (default): 266ms
2017-06-06 00-05-31 [out] onResync (default): 2656ms
2017-06-06 00-06-04 [out] onResync (default): 1019ms
2017-06-06 00-06-06 [out] onResync (default): 1854ms
2017-06-06 00-06-07 [out] onResync (default): 4ms
2017-06-06 00-06-42 [out] onResync (default): 1175ms
2017-06-06 00-07-16 [out] onResync (default): 2396ms
2017-06-06 00-07-51 [out] onResync (default): 506ms
2017-06-06 00-08-28 [out] onResync (default): 1345ms
2017-06-06 00-09-38 [out] onResync (default): 165ms
2017-06-06 00-10-17 [out] onResync (default): 494ms
2017-06-06 00-10-27 [out] onResync (default): 21ms
2017-06-06 00-11-21 [out] onResync (default): 24ms
2017-06-06 00-11-55 [out] onResync (default): 585ms
2017-06-06 00-12-31 [out] onResync (default): 669ms
2017-06-06 00-12-32 [out] onResync (default): 459ms
2017-06-06 00-12-34 [out] onResync (default): 181ms
2017-06-06 00-13-00 [out] onResync (default): 261ms
2017-06-06 00-13-44 [out] onResync (default): 957ms
2017-06-06 00-14-20 [out] onResync (default): 927ms
2017-06-06 00-14-56 [out] onResync (default): 522ms
2017-06-06 00-15-32 [out] onResync (default): 144ms
2017-06-06 00-15-37 [out] onResync (default): 121ms
2017-06-06 00-16-10 [out] onResync (default): 772ms
2017-06-06 00-16-47 [out] onResync (default): 535ms
2017-06-06 00-17-01 [out] onStateChanged (default): 1

```
how can i find out the reason for that and fix this. I want a stable playback but what i get is playback with breakouts...

thanks

**Comment by opsxcq** [comment]
I've the same problem, and everything is running locally.

**Comment by IVData** [comment]
What hardware is this running on? Server and client.

What are your snapcast configs? And what is your audio setup (pulse, alsa)?

**Comment by opsxcq** [comment]
Both server and client are running on the same machine. The server is running in a docker container, reading a FIFO from Mopidy, which is also running in a docker container. The machine is a standard I5 computer with 16gb of ram.

Operating system is Debian 9, 64bits, kernel 4.9 with pulse audio 10.0.

Everytime I pause/play, or something randomly, I got messages like these.

```
2018-02-08 20-35-32 [Info] onResync (Mopidy): 1883ms
2018-02-08 20-35-32 [Info] onStateChanged (Mopidy): 2
```
Every time the sound stops/lag this message appears.

My server command line is `snapserver -s pipe:///output/snapfifo?name=Mopidy&sampleformat=48000:16:2 -b 200`

My client command line is `snapclient -h localhost -p 1704 -d`

**Comment by badaix** [comment]
Resyncs should be fixed in v0.19
